test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -240,6 +238,44 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
     assert.equal(transactionStatus2, TransactionStatus.Succeeded)
   })
 
+  // TODO: when SDK functionality is expanded, improve test specificity by creating trustlines/issued currencies first,
+  // which will also avoid the need for maintenance after a testnet reset.
+
+  // TODO: Implement an integration test for an account with balances/assets/obligations once functionality exists for first creating things.
+  it('getGatewayBalances - account not found', async function (): Promise<
+    void
+  > {
+    this.timeout(timeoutMs)
+    // GIVEN a valid address that doesn't actually exist on the ledger
+    const walletFactory = new WalletFactory(XrplNetwork.Test)
+    const address = (await walletFactory.generateRandomWallet())!.wallet.getAddress()
+
+    // WHEN getGatewayBalances is called for that address THEN an error is propagated.
+    issuedCurrencyClient.getGatewayBalances(address).catch((error) => {
+      assert.typeOf(error, 'Error')
+      assert.equal(error, XrpError.accountNotFound)
+    })
+  })
+
+  it('getGatewayBalances - account with no issued currencies', async function (): Promise<
+    void
+  > {
+    // GIVEN a valid, funded address that has not issued any currencies
+    const wallet = await XRPTestUtils.randomWalletFromFaucet()
+    const address = wallet.getAddress()
```

Comment:
same as above

Reply:
The reason I did this here is because I imagine other integration tests *might* create trustlines using the wallet generated in the `before` (which is what we're working with rn, as opposed to a `beforeEach`) and I didn't want the *order* of tests to matter.  this will be re-evaluated in the context of all the integration tests when we take care of the outstanding Asana task to do a deep dive.